pyDeltaRCM/sed_tools.py

```diff
@@ -330,14 +383,49 @@ def _compute_Vp_ero(self, Vp_sed, U_loc, U_ero, beta):
         return (Vp_sed * (U_loc**beta - U_ero**beta) /
                 U_ero**beta)
 
-    def _limit_Vp_change(self, Vp, stage, eta, dx):
+    def _limit_Vp_change(self, Vp, stage, eta, dx, dep_ero):
         """Limit change in volume to 1/4 of a cell volume.
```

Comment:
I seem to have forgotten why this was implemented. It makes sense, but can you quickly re-explain why 1/4th was chosen? Sorryyyyy

Reply:
I'm not sure there's an explanation for why 0.25 specifically, but it does come from the original Matlab codes. The limiting is implemented to help numerical stability, so that large bed elevation changes cannot happen all at once. I guess 0.25 was just a reasonable choice.

Reply:
Interesting. thank you thank you. I didn't remember it even in the original matlab either, but it seems reasonable, like I said.